src/common/lib/string-utils.ts

```diff
@@ -5,3 +5,13 @@ export function includesIgnoringCase( string: string, substring: string ): boole
 export function replaceSpaces( string: string, replacementCharacter = '_' ): string {
 	return string.replaceAll( / /g, replacementCharacter );
 }
+
+export function tokenizeAndNormalize( string: string ) {
+	// Add more stop words as needed
+	const stopWords = new Set( [ 'a', 'an', 'and', 'the', 'in', 'on', 'at', 'of', 'this', 'to' ] );
```

Comment:
**Praise**: Ahhhh yes!! This is such a great idea, thank you for thinking ahead to this! 😁